var/spack/repos/builtin/packages/itk/package.py

```diff
@@ -26,11 +26,7 @@ class Itk(CMakePackage):
 
     version("5.3.0", sha256="57a4471133dc8f76bde3d6eb45285c440bd40d113428884a1487472b7b71e383")
     version("5.3rc02", sha256="163aaf4a6cecd5b70ff718c1a986c746581797212fd1b629fa81f12ae4756d14")
```

Comment:
Should we deprecate this release candidate now that the release is out?

Reply:
Probably; the policy on deprecation has never been clear to me.

Reply:
Official policy: https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions

Reply:
I am aware of that in the documentation. In the past, when I have proposed a deprecation based on point 3 and/or 4 from that document I have been asked to _not_ deprecate, thus leaving it unclear to me. 🤔

Reply:
Since you're maintaining this build recipe, I think it's up to you. If anyone has a problem with it they can also help maintain the recipe.